### PR TITLE
setup the project with react

### PR DESCRIPTION
**Pull request by Alzubair98**
set up the project with React , no errors while the project running

### CODE REVIEW

File: math-magicians/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```
Comment (medaminedev66) [inline math-magicians/README.md:1]: - [ ] Your readme doesn't provide enough details about the project. Please remove the boilerplate from your Readme and add some info about your project. The Readme file will give visitors a brief understanding of your project. Please use the [Microver template](https://github.com/microverseinc/readme-template) as an example.

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @Alzubair98  😃, :wave:, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

## Required Changes ♻️
- [ ] You created the react app in the "math-magicians" folder which it's not preferable because this prevents the Linters from being executed automatically in GitHub. I would recommend moving all the folders and files from the "math-magicians" folder to the root repository folder "Math-magicians".

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: math-magicians/README.md (hunk 1 above)
Comment (Alzubair98) [reply math-magicians/README.md:1]: @medaminedev66 
cool thanks for this advice i will move the files

Comment (Alzubair98) [reply math-magicians/README.md]: dear @medaminedev66 
the readme file will be updated when we merge this branch to the main branch 
thanks

File: README.md [hunk 2]
```diff
@@ -1,2 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `npm run build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (prolajumokeoni) [inline README.md:70]: - [ ] Please document your project professionally by adding a professional readme just as the previous reviewer suggested, Your readme file should have all main headers from the [microverse README template](https://github.com/microverseinc/readme-template/blob/master/README.md)  👍🏿

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @Alzubair98, 
Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
✔️ No linter errors
✔️ Git flow was correctly used

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Alzubair98) [reply README.md]: done 
thanks , i have missed that

**Review by keroloussamy — APPROVED** [review]
Hi @Alzubair98, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap

## Optional suggestions
_Check the comments under the review._

- [ ] x

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Alzubair98** [comment]
thanks man

**Pull request merged**